src/test/js/edgeCases/runDetailsDeepLink.js

```diff
@@ -30,10 +30,8 @@ module.exports = {
     },
     /** Check Job Blue Ocean Pipeline Activity Page has run */
     'Step 03': function (browser) {
-        const blueActivityPage = browser.page.bluePipelineActivity().forJob(jobName, 'jenkins');
```

Comment:
You remove some "assurance code" so the job is in a certain state. Are you sure?